src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+
+I was a dedicated Xbox user at the time, for everything besides gaming. I diligently connected to #xbins on EFnet to find the new binary release and watched in awe the quick iteration these folks were doing, adding so many incredible features, even supporting things like CD+G for Karaoke. You have to realise how groundbreaking a library mode was, allowing you to peruse your stored media collection while everyone else - even Netflix - was still renting DVDs by post or picking them up in strip mall stores. 
+
+The developers themselves deserve a ton of recognition: their tireless pursuit of the best media experience, one that was infinitely customizable and powerful and something they were proud of. While most have moved on to other things, a few, like **spiff**, remain around since those early days, and are still here to let others know why certain things are why they are and even to still occasionally contribute code.
+
+**Success Has Many Parents**
+
+But these folks did not do this in a vacuum: as the saying goes, "it takes a village". [FFmpeg](https://ffmpeg.org), for example, being the Internet’s Swiss Army knife for video, handles so much of the core rendering and playback. In developer parlance, we call these ‘dependencies’ but I prefer to think of these types of library as the backbone. Without the ability to stand on the shoulder of these mammoth libraries , XBMC devs would never have been able to focus so much on the skinning and user experience. While XBMC was initially heavily based on [mplayer](http://www.mplayerhq.hu), which is now known as [mpv](https://mpv.io), it has evolved into being closely tied to FFmpeg. 
+
+![XBMC Home Screen, 2009](/images/blog/xbmc-2009-1080p.webp "Image of the 2009 XBMC home screen and menu")
+
+There are also countless other contributors - to the code, artwork, forum, and beyond - who absolutely have to be acknowledged. Today's Kodi wouldn't be what it is without the add-on ecosystem, translators, community moderators, volunteer artists, system admins, database admins, and many more.
+
+And while devs have came and gone, the project has lived on, continuing to expand to every major platform it possibly can, even to a couple of smart TV ones now. Highly unusual - maybe unique - as an open source application that started on a Microsoft product before spreading further to so many other, disparate platforms, Kodi has now even come full circle to be back in the [Xbox Store](https://apps.microsoft.com/detail/9NBLGGH4T892?hl=en-US&gl=US) (although, somewhat ironically, Microsoft platforms have become increasingly challenging for us because we lack current developer knowledge). We still try to bring all the functionality we can to every platform we can: you've got a weird mod file, from your demoscene days? Still got a pile of FLVs? We got you. What about wanting to stream Widevine-enabled Internet content with an open source project? Yeah, we see you and we care.
```

Comment:
```suggestion
And while devs have came and gone, the project has lived on, continuing to expand to every major platform it possibly can, even to a couple of smart TV ones now. Highly unusual - maybe unique - as an open-source application that started on a Microsoft product before spreading further to so many other, disparate platforms, Kodi has now even come full circle to be back in the [Xbox Store](https://apps.microsoft.com/detail/9NBLGGH4T892?hl=en-US&gl=US) (although, somewhat ironically, Microsoft platforms have become increasingly challenging for us because we lack current developer knowledge). We still try to bring all the functionality we can to every platform we can: you've got a weird mod file, from your demoscene days? Still got a pile of FLVs? We got you. What about wanting to stream Widevine-enabled Internet content with an open-source project? Yeah, we see you and we care.
```

Reply:
Adjective again, so corrected.